scripts/setup.sh

```
@@ -138,6 +99,16 @@ pip install -r requirements.txt
 deactivate
 cd ..
 
+# ---- Set up the sync-microservice ----
+echo -e "${YELLOW}Setting up sync-microservice...${NC}"
+cd sync-microservice
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)